builtin/game/item.lua

```diff
@@ -605,6 +609,7 @@ core.craftitemdef_default = {
 	-- Interaction callbacks
 	on_place = redef_wrapper(core, 'item_place'), -- core.item_place
 	on_drop = redef_wrapper(core, 'item_drop'), -- core.item_drop
+	on_rightclick = redef_wrapper(core, 'item_rightclick'),
```

Comment:
The function is titled on_rightclick in entities, otherwise I would've gone with on_right_click